src/utils/validate-token.ts

```diff
@@ -0,0 +1,20 @@
+import { AuthError } from "./error-handling";
+import jwt from "jsonwebtoken";
+
+export const validateToken = (token: string) => {
+  const isThereToken = token;
+  if (!isThereToken) {
```

Comment:
Acho que ficaria claro o suficiente sem a variável:

```suggestion
  if (!token) {
```

O que acha?

Reply:
Melhor mesmo